ci/scripts/update-mainnet-artifacts.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# This script is run by the update-mainnet-artifacts job in the ci-pr-only workflow
+# to update ./mainnet-artifacts.bzl bazed on the contents of testnet/mainnet_revisions.json.
+# The latter tracks the IC version (git revision) of the mainnet NNS subnet (with principal tdb26-...).
+#
+# This script will set the MAINNET_NNS_SUBNET_IC_VERSION bazel variable and will compute the SHA256 hashes
+# of the PUBLISHED_BINARIES which are using the MAINNET_NNS_SUBNET_IC_VERSION.
+
+set -eufo pipefail
+
+EXIT_STATUS=0
+
+# Retrieve the IC version (git revision) of the mainnet NNS subnet:
+MAINNET_NNS_SUBNET_ID="tdb26-jop6k-aogll-7ltgs-eruif-6kk7m-qpktf-gdiqx-mxtrf-vb5e6-eqe"
+MAINNET_NNS_SUBNET_IC_VERSION="$(jq ".subnets.\"$MAINNET_NNS_SUBNET_ID\"" -r testnet/mainnet_revisions.json)"
```

Comment:
would suggest using `jq --arg subnetid "$MAINNET..." '.subnets.[$subnetid]...'`

Reply:
Oh that's nice. Will test it out.

Reply:
Done.